app/views/achievements/manage.html.haml

```diff
@@ -1,18 +1,18 @@
 .row
   .col-xs-12.table-topper
     %h2= t(:achievements)
-    = link_to new_achievement_path, class: ["btn", "btn-success"] do
-      %span.glyphicon.glyphicon-plus
-      = t(:add_achievement)
 .row
   .col-xs-12
-    %table.table.achievements
+    %table.table.achievements.table-condensed
       %tr
         %th #
         %th= t(:achievement_title)
         %th= t(:achievement_side)
         %th= t(:achievement_points)
         %th
+          = link_to new_achievement_path, class: ["btn", "btn-success"] do
+            %span.glyphicon.glyphicon-plus
+            = t(:add_achievement)
```

Comment:
Moved the `Add Achievement` button to the very right and insde the table header.

This freed up some vertical space and pulled the button into the same column as the other buttons.